### ISSUE TITLE
accessibility issue 16 (medium) - more descriptive alt text for splash images

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#241]
The alt text used for the linked images isn’t descriptive enough. Images that are used as links should have alt text that is describing the destination of the link.  In the case of the carousel, there are captions below the image, and the caption text should be included in the alt text for users to better understand where the link is going.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/f4d808714cd99f93fe6d37eccfef699936180915
